src/main/java/seedu/pill/util/Printer.java

```diff
@@ -28,9 +28,17 @@ public static void printSpace(){
 
     /**
      * Initializes the bot, prints the logo and a welcome message.
+     *
+     * @param items Reference ItemMap to print restock list.
+     * @param threshold The minimum number of items before it is deemed to require replenishment.
      */
```

Comment:
Should the javadoc be updated to reflect that the method prints items to be restocked?